### DISCUSSION TITLE
Eviller Magit

### DISCUSSION THREAD

**Discussion by Kazark** [syl20bnr/spacemacs#14727] [Q&A] [answered]
I have been using Spacemacs with Magit for 3 or 4 years now (I'm loosing track) after 10 years of Vim and my fingers have never learned that Magit is not Evilified the same way as regular textual buffers. Keys like `$` are bound with special meaning. I wonder if Doom has a more evilified Magit experience, and if we might provide a switch with an optional set of keybindings for that.

Any advice? Anyone also stumble over some of the basic textual motion keys being bound with other semantics in Magit? If I remember, the lack of visual mode is another one of the things that trips me up (it's actually hard for me to remember what they all are, since they are more finger memory than anything).

**Comment by lebensterben** [comment, marked as answer]
Take a look here: https://github.com/emacs-evil/evil-collection/tree/master/modes/magit

And what else are you missing?

**Reply by Kazark** [reply]
`evil-collection-magit-use-y-for-yank` 💯

**Reply by Kazark** [reply]
I forget though, what is the status of using `evil-collection` with Spacemacs? Last I looked into it, they weren't compatible.

**Reply by Kazark** [reply]
Toggle text mode---that seems like a really good idea.

**Reply by lebensterben** [reply]
There are a few packages merged into `evil-collection`.
What is exactly not compatible now?

---

> Toggle text mode---that seems like a really good idea.

Agree. If you want you can make a PR to introduce new binding this in layer README.org.
This is really handy when reviewing changes for example.

**Reply by Kazark** [reply]
Oh wow, I was confused. Spacemacs actually loads `evil-collection` in the `spacemacs-evil` layer.

**Reply by Kazark** [reply]
Excellent, I just set `evil-collection-magit-use-y-for-yank` to `t` and it works, without changing anything else.
